requirements/tox_requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Packages needed in order to run tox.
+# Installed by Travis and dev environments.
+tox==2.5.0
```

Comment:
I gave `pip-tools` a shot, but it can't resolve the url needed for our special version of `fs`. [More info](https://github.com/nvie/pip-tools/pull/405). For the time being I'm going to add hand-edited requirements files that match what pip-tools compiles. However I'm not going to include the `.in` files since regenerating using them won't work, and could break things. When we graduate to using fs2 this can be more sane.